<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Create">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>341</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new order</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>341</width>
      <height>240</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>requisites</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>order_number</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>price_of_one</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="requisites"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>order_date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="order_number"/>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="price_of_one"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="product_code">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="product_name"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>product_name</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>product_code</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="order_date"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="quantity_of_goods"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>quantity_of_goods</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="add_product">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>351</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>add product</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finish_additing">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>351</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>finish adding</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="show_result">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>351</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>show table</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="display">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>631</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="open_file">
       <property name="text">
        <string>Open file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_file">
       <property name="text">
        <string>Resave to file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_in_file">
       <property name="text">
        <string>Show in file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="read_orders">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>341</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Read orders from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="read_items">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>341</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Read item from file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Create</sender>
   <signal>released()</signal>
   <receiver>order_number</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Create</sender>
   <signal>released()</signal>
   <receiver>order_date</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Create</sender>
   <signal>released()</signal>
   <receiver>requisites</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Create</sender>
   <signal>released()</signal>
   <receiver>product_code</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Create</sender>
   <signal>released()</signal>
   <receiver>quantity_of_goods</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Create</sender>
   <signal>released()</signal>
   <receiver>price_of_one</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Create</sender>
   <signal>released()</signal>
   <receiver>product_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_product</sender>
   <signal>released()</signal>
   <receiver>product_code</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_product</sender>
   <signal>released()</signal>
   <receiver>quantity_of_goods</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_product</sender>
   <signal>released()</signal>
   <receiver>price_of_one</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_product</sender>
   <signal>released()</signal>
   <receiver>product_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
